README.md

```diff
@@ -1,2 +1,162 @@
-# asgardeo-nextjs-b2b-sample-app
-Next.JS sample web application to integrate with Asgardeo for B2B usecases.
+## 👀 Live Deployment
+
+A live preview of this demo is available at
+
+1. Vercel: [https://asgardeo-b2b-nextjs-sample-app.vercel.app/](https://asgardeo-b2b-nextjs-sample-app.vercel.app/)
+
+Sample Credential
+
+- organization name : Best Car Mart
+- username : Alex@gmail.com
+- password : Alex@123
+
+## Features/ Capabilities
+
+- Sign In to the sub organization application
+- Sign Out of the application
+- User profile view and update
+- User Management.
+  - View all users
+  - Add new user
+  - Edit user
+  - Delete User
+- Group Management
+  - View groups
+  - Add a new group and assign users.
+  - Edit group and edit assigned users.
+  - Delete group
+- Role Management
+  - View Role permission
+  - View assigned groups
+  - Assign new groups
+  - Assign external groups
+- Idp Management
+  - Add Idp ( Google Idp/ Enterprise Idp)
+  - Edit Idp settings
+  - Add external groups
+  - Add Idp to application login flow
+
+## Configure Asgardeo to integrate with your application
+
+Create a sub organization
+
+1. Access Asgardeo at https://console.asgardeo.io/ and log in.
+
+2. Click Sub Organizations in the left navigation menu.
+
+3. Create a new sub organization
+   <img src="./images/docs/subOrg.png" alt="SubOrg" title="SubOrg">
+
+Create the application
+
+1. Click Applications in the left navigation menu.
+
+2. Create a new application
+
+- Select Standard-Based Application.
```

Comment:
Let's link the corresponding docs whenever possible

Reply:
fixed